llorens/day09/Day09Solver.py

```diff
@@ -0,0 +1,66 @@
+def is_valid_char(j, sequence):
+    valid = True
+
+    for i in range(j-1, -1, -1):
```

Comment:
What? What kind of range is this generating?

Reply:
From j-1 to -1 and each iteration decreasing i by 1.

Range(From, To, Step)